nbs/models.ipynb

```diff
@@ -57,6 +57,7 @@
     "import pandas as pd\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            return f'HistoricAverage()'
</pre><p>Please remove the parenthesis after <code>HistoricAverage</code> (it was a previous change now fixed in main). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>